src/TopBar.css

```
@@ -37,6 +37,101 @@
+.city-picker__search--desktop.ant-btn {
+  min-width: 280px;
+  justify-content: flex-start;
+  text-align: left;
+  font-weight: 400;
+}
+
+/* Desktop topbar map actions: shared size (solid fills read heavier than the glass
+   search field, so a hair smaller for optical balance). Colors stay per-button. */
+.topbar-map-action.ant-btn {
+  --topbar-map-action-size: calc(var(--ant-control-height, 32px) - 2px);
+  width: var(--topbar-map-action-size);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)